tests/requirements-libraries.txt

```diff
@@ -54,6 +54,7 @@ Pillow==4.2.1
 pycryptodomex==3.4.6
 future==0.16.0
 pyusb==1.0.0
+requests==2.15.1 # Newest version of requests that allows tests to pass
```

Comment:
The newest version of requests that packages the urllib3 library for the requests_urllib_six test is 2.15.1. All newer versions only require urllib3 and the test fails due to other issues with PyInstaller not being able to package the required libraries correctly.